src/microbe_stage/MicrobeSpecies.cs

```diff
@@ -36,6 +36,10 @@ public override string StringCode
     [JsonIgnore]
     public float BaseSpeed => MicrobeInternalCalculations.CalculateSpeed(Organelles, MembraneType, MembraneRigidity);
 
+    [JsonIgnore]
+    public float BaseSize => Organelles.Organelles.Sum(organelle => organelle.Definition.HexCount)
```

Comment:
I think this is too general. This should be engulf size? Or at least like "EffectiveTotalHexSize" right now this "size" is way too general to me as to what this is related to (because my first thought about size would be how large the cell is visually and that is more connected to the membrane radious).